Bust Config.rb

```diff
@@ -155,7 +163,31 @@
     face_name: "Yarra emo2",
     face_indexes: [3],
   },
+  # Topless young Simon needs a special beardless bust
+  {
+    character_name: "Simon topless young",
+    face_name: "face002b topless",
+    face_indexes: [7],
+  },
+  {
+    character_name: "Simon topless young",
+    face_name: "face002b topless dark eyes",
+    face_indexes: [7],
+  },
```

Comment:
I originally did map it to nil, but it would cause a game-crashing 'file not found' error if it ever was triggered either intentionally or accidentally. Decided to err on the safe side.

Reply:
... Oh, got it. It's because `custom_bust_display_options`, being very much a self-hack, bypasses most security checks.